front/app/components/ProjectAndFolderCards/ProjectAndFolderCards.tsx

```diff
@@ -51,26 +51,17 @@ const ProjectAndFolderCards = ({
   statusCounts,
   showTitle,
   layout,
-  publicationStatusFilter,
+  publicationStatusFilters,
   onChangeAreas,
   onChangeTab,
 }: Props) => {
   const adminPublications = useAdminPublications({
     pageSize: 6,
-    publicationStatusFilter,
+    publicationStatusFilters,
     rootLevelOnly: true,
     removeNotAllowedParents: true,
   });
 
-  const publicationStatusesStringified = JSON.stringify(
-    publicationStatusFilter
-  );
-
-  useEffect(() => {
-    adminPublications.onChangePublicationStatus(publicationStatusFilter);
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [publicationStatusesStringified]);
-
   const availableTabs = useMemo(() => {
```

Comment:
Doing this in useAdminPublications now.

Reply:
See my comment in `front/app/hooks/useAdminPublications.ts`